specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1138,7 +1138,7 @@ Connector architecture.
 The JDBC 4.2 specification is available at
 _https://jcp.org/en/jsr/detail?id=221_ .
 
-===== [[a2553]]Jakarta™ API for XML Web Services (JAX-WS) Requirements
+===== [_Jakarta™ XML Web Services (JAX-WS) 2.3 Requirements
```

Comment:
do not remove any of the \[[xxxx]] items.  These are links used by references \<<..>> in other parts of the doc.